lib/msf/util/exe.rb

```diff
@@ -1437,6 +1437,57 @@ def self.to_powershell_hta(framework, arch, code)
     read_replace_script_template("to_powershell.hta.template", hash_sub)
   end
 
+  def self.to_python_reflection(framework, arch, code, exeopts)
+    if arch != ARCH_X64 and arch != ARCH_X86
+      raise RuntimeError, "Msf::Util::EXE.to_python_reflection is not compatible with #{arch}"
+    end
+    python_code = 'import ctypes,os'
+    load_socket_nt = ''
+    load_socket_posix = ''
+    payload = exeopts[:payload]
+    if payload.end_with?('meterpreter/reverse_tcp')
```

Comment:
Is this chunk needed? Without passing in the `LHOST` and `LPORT` wouldn't the native `reverse_tcp` stager do the connection and staging itself? I think we don't have to keep the socket operations for this, just the reflection part.

Reply:
It's not essential but it's useful because it means the output is cross-platform. e.g the met.py generated by all the commands in the verification list is the same, so you can just swap the handler depending on which architecture you're targetting.
Happy to change it (it makes some things simpler) if you think it's not useful. It would probably be more useful if there was a multi/meterpreter/reverse_tcp handler.